payments/utils.py

```diff
@@ -366,6 +366,9 @@ def send_payment_notification(order, request, email=None):
     if email is None:
         email = order.customer_email
 
+    if not email:
+        raise VenepaikkaGraphQLError(_("No email was found"))
```

Comment:
Can you replace this with `ValidationError`? Since it's not specifically a GraphQL error

Reply:
ok I copied your check from https://github.com/City-of-Helsinki/berth-reservations/pull/455/files so it won't conflict when you rebase